contributors/design-proposals/storage/csi-migration.md

```diff
@@ -322,19 +372,36 @@ And at this point users doing their own deployment and not installing the GCE PD
 CSI driver encounter an error.
 
 ## Testing
+
 ### Standard
+
 Good news is that all “normal functionality” can be tested by simply bringing up
 a cluster with “migrated” drivers and running the existing e2e tests for that
 driver. We will create CI jobs that run in this configuration for each new
 volume plugin
 
 ### Migration/Non-migration (Upgrade/Downgrade)
+
 Write tests were in a normal workflow of attach/mount/unmount/detach, we have
 any one of these operations actually happen with the old volume plugin, not the
 CSI one This makes sure that the workflow is resiliant to rollback at any point
 in time.
 
 ### Version Skew
+
 Master/Node can have up to 2 version skw. Master must always be equal or higher
 version than the node. It should be covered by the tests in
 Migration/Non-migration section.
+
+### Scalability & Performance
+
+Migration adds some overhead in the form of API calls, a new RPC boundary,
+driver latency, and shim latency. It is important that we document the current
+performance of in-tree drivers then test and document those drivers with
+migration. The performance hit must be within acceptable ranges TBD. We define
+"volume lifecycle operations" as including
+attach/detach/mount/unmount/mountdevice/unmountdevice/resize/etc. Important
+scalability & performance metrics include but are not limited to:
+* Supported number of concurrent volume lifecycle operations
```

Comment:
@msau42 PTAL, feel like there should be more but these 2 seemed like the most important